cpp/include/cudf_test/iterator_utilities.hpp

```diff
@@ -77,9 +75,9 @@ template <typename Iter>
  * @param indices The indices for which the validity iterator must return `false` (i.e. null)
  * @return auto Validity iterator
  */
-[[maybe_unused]] static auto iterator_with_null_at(cudf::host_span<cudf::size_type const> indices)
+[[maybe_unused]] static auto nulls_at(std::vector<cudf::size_type> const& indices)
```

Comment:
When I first wrote it, this API used to take a `std::vector`. The suggestion at the time was that `host_span`s are preferable.

I agree with your current change. (Personally, I prefer this be `vector<>` so that call sites don't have to explicitly construct vectors.) I hope this is agreeable with @harrism  and @vuule.

Reply:
Yeah, with `host_span` parameter, you cannot call `nulls_at({1, 2, 3})`. You must call `nulls_at(std::vector<int>{1, 2, 3})`. This is so much inconvenient.